index.html

```diff
@@ -0,0 +1,6349 @@
+<!Doctype html>
+<html>
+<title>Canvas Palette</title>
+<meta charset='utf-8'>
+<head>
+    <title>imgExample</title>
+    <script src="data/4x4.json"></script>
+    <script src="data/32x32.json"></script>
+    <link href="style.css" rel="stylesheet"/>
+</head>
+<body>
+<header>
+    <h1>CodeJam - Pallete</h1>
+</header>
+<main>
+    <div>
+        <div class="menu">
+        <section class="section-1">
+            <ul>
+                <li>Paint bucket</li>
+                <li>Choose color</li>
+                <li>Move</li>
+                <li>Transform</li>
+            </ul>
+        </section>
+        <section class="section-2">
+            <ul>
+                <li>Current color</li>
+                <li>Prev color</li>
+                <li>red</li>
+                <li>blue</li>
+            </ul>
+        </section>
+    </div>
+    <div>
+        <form>
+            <p>Please select size:</p>
+            <input type="radio" name="picture" value="32x32" onclick="handleClick(this);">32x32<br>
+            <input type="radio" name="picture" value="4x4" checked="checked" onclick="handleClick1(this);">4x4<br>
+            <input type="radio" name="picture" value="256x256" onclick="handleClick2(this);">256x256<br>
+        </form>
+
+
+        <canvas class="picture1" width="512" height="512" style = "border:2px solid grey"></canvas>
+        <canvas class="picture2" width="512" height="512" ></canvas>
+        <canvas class="picture3" width="512" height="512" ></canvas>
+    </div>
+    </div>
+</main>
+
+<script>
+    let pic3 = document.getElementsByClassName("picture3");
+function handleClick(){
+   document.getElementsByClassName("picture3")[0].style.display="none";
```

Comment:
if you want to find element using class, use querySelector or querySelectorAll

Reply:
it's a modern way for finding elements